### DISCUSSION TITLE
Larger App structure: Lack of Example

### DISCUSSION THREAD

**Discussion by davidpham87** [day8/re-frame#685] [Q&A] [answered]
Hello,

I hope this goes into the right place, if not, please excuse me. According to the documentation in App structure, larger apps should have hierarchical structure in their folder, where basically each subfolder creates the recommended file structure for a simple app. However, I could not find a project in the external resource that follows the guideline. 

Would it be possible to find an example? Or maybe adapt the docuementation?

Best regards,
David

**Comment by mike-thompson-day8** [comment, marked as answer]
The docs already provide examples: 
https://github.com/Day8/re-frame/blob/master/docs/External-Resources.md

**Comment by davidpham87** [comment]
I agree the documentation provides examples of apps, but most don’t follow the structure proposed by https://github.com/Day8/re-frame/blob/master/docs/Basic-App-Structure.md.
